js/status.js

```diff
@@ -97,17 +97,21 @@ class Status {
 				total: -1,
 				spf_aligned: 0,
 				dkim_aligned: 0,
-				dkim_spf_aligned: 0
+				dkim_spf_aligned: 0,
+				quarantined: 0,
+				rejected: 0
```

Comment:
I believe the six blocks in the status bar are too many for a quick look. Maybe it will be better to replace dkim_aligned and dkim_spf_aligned with quarantined and rejected or just add an option to choose what to display?

Reply:
Like in the the screenshot - it fits on my laptop. I was aiming for a kind of green to red gradient. The more to your right in the blocks, the more in trouble you are. 

For a quick view, the alignedness could maybe be optional, it kind of depends on what part of DMARC your focusing on. Debugging non-delivered messages or monitoring the overall performance. 

<img width="1672" alt="Screenshot 2023-03-16 at 21 18 51" src="https://user-images.githubusercontent.com/11736695/225743542-7e329fbc-aa9d-4b3a-9ffe-e3f19a87b647.png">